app/views/events/_invite_form.html.erb

```diff
@@ -0,0 +1,26 @@
+<% if current_user.id == @event.user_id %>
+  <%= render 'partials/form_errors', resource: @event %>
+  <div class="row center-xs middle-xs mt-5">
+    <div class="col-md-6 col-sm-8 col-xs-12">
+      <div class="row start-xs">
+        <div class="window w-100">
+          <div class="title-bar">
+            <div class="title-bar-text">Internet explorer says hi</div>
+          </div>
+          <div class="window-body">
+            <h4>Invite new Attendee</h4>
+            <%= form_with(url: invite_to_event_path, method: "post") do %>
+              <div class="field-row-stacked">
+                <%= label_tag("user_name", "Username to Invite") %>
+                <%= text_field_tag("user[name]") %>
+              </div>
+              <div class="flex justify-end mt-3">
+                <button>Invite</button>
+              </div>
+            <% end %>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+<% end %>
```

Comment:
- [ ] put the if statement in a partial that renders a partial containing the markup within the if statement so you don't have this complexity directly in the partial